### ISSUE TITLE
Lecterns don't show tabs

### ISSUE THREAD

**Issue by mlv** [Andrew6rant/InventoryTabs#62]
Lecterns are recognized as having an inventory, so show up in the tabs. But when you go to a lectern, the tabs are gone and you can't go to anything else.

**Comment by Andrew6rant** [comment]
Yeah, lecterns only show up as an inventory tab when they have a book in them. The book GUI is kinda funky, I haven't yet mixed into it to get the tabs to render there
